packages/router/src/controllers/Rest/Swagger.ts

```diff
@@ -69,12 +67,7 @@ export class SwaggerGenerator {
           },
         },
       },
-      security: [
-        {
-          clientid: [],
-          clientSecret: [],
-        },
-      ],
+      security: JSON.parse(JSON.stringify(this._routerMetadata.globalSecurityHeaders)),
```

Comment:
We could also entirely avoid repeated stringifications by just passing a pre-stringified `globalSecurityHeaders` instead. Totally type hacky, but definitely worth it perf-wise.

Reply:
Ended up stringifying once per Swagger construction instead (that makes 2 times total).